### ISSUE TITLE
Crashing out game with latest update: 2189: Cayo Perico Heist

### ISSUE THREAD

**Issue by waynieoaks** [alexguirre/CScenarioPoint-Patch#9]
Hi

It looks like the latest update: 2189: Cayo Perico Heist causes this to no longer work and game crashes out during loading. 

Many thanks.

**Comment by alexguirre** [comment]
Yeah, latest update broke it. But R* increased the model set limit in scenario points to 512 from 256, so you may no longer require this patch.
